nutils/evaluable.py

```diff
@@ -923,7 +923,7 @@ def _node(self, cache, subgraph, times):
     if self in cache:
       return cache[self]
     args = tuple(arg._node(cache, subgraph, times) for arg in self._Evaluable__args)
-    label = '\n'.join(filter(None, (type(self).__name__, self._node_details, ','.join(map(repr, self._axes)))))
+    label = '\n'.join(filter(None, (type(self).__name__, self._node_details, ':'.join(filter(None, (self.dtype.__name__[0], ','.join(map(repr, self._axes))))))))
```

Comment:
So this is where I personally would just keep the `:` regardless of dimension. In fact - and I think @joostvanzwieten should weight in here - I actually fail to see why we don't just use `label = str(self)` in the `Evaluable._node` base implementation so we don't have to redefine it in `Array`, and automatically get consistency.